saleor/order/fetch.py

```diff
@@ -107,13 +116,28 @@ def get_manual_line_discount(
 
 
 def fetch_draft_order_lines_info(
-    order: "Order", lines: Iterable["OrderLine"] | None = None
+    order: "Order",
+    lines: Iterable["OrderLine"] | None = None,
+    extend: bool = False,
 ) -> list[EditableOrderLineInfo]:
+    """Fetch the necessary order lines info in order to recalculate its prices.
+
+    `extend` argument determines if the function should additionally retrieve the latest
+    variant channel listing prices
```

Comment:
Please use some more meaningful name, like `prefetch_channel_prices`, or something different that explain what is this flag used for